pkg/scaffold/scaffold.go

```diff
@@ -53,7 +53,7 @@ type Scaffold struct {
 	BoilerplateOptional bool
```

Comment:
when is used the BoilerplateOptional?

Reply:
`Scaffold.Execute` or `Scaffold.defaultValues` IIRC.